### PR TITLE
New Version: Tonec.InternetDownloadManager 6.41.7

### PR DESCRIPTION
**Pull request by brotich**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/99194)

### CODE REVIEW

**Comment by brotich** [comment]
This PR fixes https://github.com/microsoft/winget-pkgs/pull/98907. The winget bot is adding spaces to the version number

File: manifests/t/Tonec/InternetDownloadManager/6.41.7.1/Tonec.InternetDownloadManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Tonec.InternetDownloadManager
+PackageVersion: 6.41.7.1
```
Comment (mdanish-kh) [inline manifests/t/Tonec/InternetDownloadManager/6.41.7.1/Tonec.InternetDownloadManager.installer.yaml:4]:
```suggestion
PackageVersion: 6.41.7
```
Must match Control Panel / Apps&Features

File: manifests/t/Tonec/InternetDownloadManager/6.41.7.1/Tonec.InternetDownloadManager.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Tonec.InternetDownloadManager
+PackageVersion: 6.41.7.1
```
Comment (mdanish-kh) [inline manifests/t/Tonec/InternetDownloadManager/6.41.7.1/Tonec.InternetDownloadManager.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 6.41.7
```
Must match Control Panel / Apps&Features

File: manifests/t/Tonec/InternetDownloadManager/6.41.7.1/Tonec.InternetDownloadManager.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Tonec.InternetDownloadManager
+PackageVersion: 6.41.7.1
```
Comment (mdanish-kh) [inline manifests/t/Tonec/InternetDownloadManager/6.41.7.1/Tonec.InternetDownloadManager.yaml:4]:
```suggestion
PackageVersion: 6.41.7
```
Must match Control Panel / Apps&Features

File: manifests/t/Tonec/InternetDownloadManager/6.41.7.1/Tonec.InternetDownloadManager.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /skipdlgs
+  SilentWithProgress: /skipdlgs
+UpgradeBehavior: install
+ReleaseDate: 2022-12-03
```
Comment (mdanish-kh) [inline manifests/t/Tonec/InternetDownloadManager/6.41.7.1/Tonec.InternetDownloadManager.installer.yaml:17]:
```suggestion
ReleaseDate: 2023-02-27
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
After committing the suggested changes, you would also have to rename the version folder to match the updated PackageVersion. The folder under `manifests/t/Tonec/InternetDownloadManager` should be renamed from `6.41.7.1` -> `6.41.7`

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required folder and make the appropriate rename changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**